module_fv3_config.F90

```diff
@@ -25,6 +26,7 @@ module module_fv3_config
   logical                  :: force_date_from_configure
   logical                  :: restart_endfcst
 !
+  real,dimension(:),allocatable                   :: output_fh
```

Comment:
I think this array (`output_fh`) should be an integer array storing seconds, not real array storing hours. The elements of this array are compared for equality to `nfseconds` or `seconds`, and both of them are integers. Then there will be no need to have multiplication by 3600 or division by 3600 at so many places. As soon as configuration parameters `nfhout`, `nfhmax_hf` and  `nfhout_hf` are read in they can be converted to seconds and all arithmetic should be done with integers (seconds) only. Which will simplify the code.

Reply:
For history reasons, this array is storing forecast hours (especially for climate runs) to maintain readability. In the code, so far we are comparing forecast time in seconds. But In the future, if finer time available, this can be extended to compare time in real seconds. So I think it's good to keep output_fh for forecast hour as real number, not integer seconds